_docs-2/troubleshooting/zookeeper.md

```diff
@@ -3,6 +3,34 @@ title: ZooKeeper
 category: troubleshooting
 order: 7
 ---
+## ZooKeeper ACLs
+
+Accumulo requires full access to nodes in ZooKeeper under the /accumulo path.  The ACLs can be examined using the
+ZooKeeper cli `getAcl` and modified with `setAcl` commands.  With 2.1.1, the zoo-info-viewer utility has an option
+that will print all of the ACLs for the nodes under `/accumulo/[INSTANCE_ID]` (See the [zoo-info-viewer]).  
+To run the utility, only ZooKeeper needs to be running. If hdfs is running, the instance id can be read from hdfs, 
+or it can be entered with the zoo-info-viewer command --instanceId option.  Accumulo management processes 
+*do not* need to be running. This allows checking the ACLs before starting an upgrade. 
+
```

Comment:
Might be useful to include a one or two line example output.

Reply:
Added in 12938808